### PR TITLE
chore: generic subxt signer

### PR DESCRIPTION
**Pull request by alstjd0921**
## Description

Please include a summary of the changes you request.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Something else (simple changes that are not related to existing functionality or others)

# Checklist

- [ ] I have selected the correct base branch.
- [ ] I have performed a self-review of my own code.
- [ ] I have made corresponding changes to the documentation.
- [ ] I have made new test codes regarding to my changes.
- [ ] I have no personal secrets or credentials described on my changes.
- [ ] I have run `cargo-clippy`  and linted my code.
- [ ] My changes generate no new warnings.
- [ ] My changes passed the existing test codes.
- [ ] My changes are able to compile.

### CODE REVIEW

File: client/src/btc/handlers/inbound.rs [hunk 1]
```diff
@@ -151,18 +152,24 @@ impl<T: JsonRpcClient + 'static> InboundHandler<T> {
...
 		self.bfc_client.clone()
 	}
 }
 
 #[async_trait::async_trait]
-impl<T: JsonRpcClient + 'static> Handler for InboundHandler<T> {
+impl<T: JsonRpcClient + 'static, S: Signer<CustomConfig>> Handler for InboundHandler<T, S>
+where
+	S: Send,
+	S: Sync,
```
Comment (dnjscksdn98) [inline client/src/btc/handlers/inbound.rs:171]:
```suggestion
    T: JsonRpcClient,
	S: Signer<CustomConfig> + Send + Sync,
```
where 절에 다 넣든, impl에 넣든 하나로 통일하시죠

Comment (dnjscksdn98) [reply client/src/btc/handlers/inbound.rs:171]: 다른 곳들도 마찬가지

File: client/src/btc/handlers/mod.rs [hunk 2]
```diff
@@ -12,6 +12,8 @@ use crate::{
 	eth::EthClient,
 };
 
+use super::block::EventMessage;
+use br_primitives::substrate::CustomConfig;
 use br_primitives::{
 	bootstrap::BootstrapSharedData,
 	eth::{BootstrapState, GasCoefficient},
```
Comment (dnjscksdn98) [inline client/src/btc/handlers/mod.rs:19]: 여기는 use 병합하구요

**Review by dnjscksdn98 — APPROVED** [review]

File: client/src/btc/handlers/mod.rs (hunk 2 above)
Comment (alstjd0921) [reply client/src/btc/handlers/mod.rs]: https://github.com/bifrost-platform/bifrost-relayer.rs/pull/151/commits/d1d14806074d3d7c5871df82069328d7a115b4ba

File: client/src/btc/handlers/inbound.rs (hunk 1 above)
Comment (alstjd0921) [reply client/src/btc/handlers/inbound.rs]: https://github.com/bifrost-platform/bifrost-relayer.rs/pull/151/commits/8d35d44a8091404098f34b690b6cc13854773175

**Pull request merged**